components/brave_vpn/common/brave_vpn_utils.cc

```diff
@@ -84,6 +85,22 @@ bool IsBraveVPNEnabled() {
          base::FeatureList::IsEnabled(skus::features::kSkusFeature);
 }
 
+std::string GetBraveVPNEntryName(version_info::Channel channel) {
+  const std::string entry_name(kBraveVPNEntryName);
```

Comment:
I think we have to make `kBraveVPNEntryName` internal constant for this function because it can be mistakenly used by anyone from the header.

Reply:
Done.